schemas/browsers-schema.md

```diff
@@ -80,7 +80,7 @@ The release objects consist of the following properties:
 This structure is exported for consumers of `@mdn/browser-compat-data`:
 
 ```js
-> const compat = require('@mdn/browser-compat-data');
-> compat.browsers.firefox.releases['1.5'].status;
+> import bcd from '@mdn/browser-compat-data';
+> bcd.browsers.firefox.releases['1.5'].status;
```

Comment:
This looks like Node.js REPL, which doesn't seem to support import statements yet:

```console
% node
Welcome to Node.js v16.15.0.
Type ".help" for more information.
> import bcd from '@mdn/browser-compat-data';
import bcd from '@mdn/browser-compat-data';
^^^^^^

Uncaught:
SyntaxError: Cannot use import statement inside the Node.js REPL, alternatively use dynamic import
```

Maybe it would make sense to describe both CommonJS and Module ways of accessing it?